tests/conftest.py

```diff
@@ -87,16 +88,30 @@ def pytest_collection_modifyitems(session, config, items):
 
     """
     for item in items[:]:
-        skip_marker = item.get_closest_marker("skipif_ocs_version")
-        if skip_marker:
-            skip_condition = skip_marker.args
-            # skip_condition will be a tuple
-            # and condition will be first element in the tuple
-            if skipif_ocs_version(skip_condition[0]):
-                log.info(
-                    f'Test: {item} will be skipped due to {skip_condition}'
-                )
-                items.remove(item)
+        skip_markers = [
+            item.get_closest_marker("skipif_ocs_version"),
+            item.get_closest_marker("skipif_lso_deployment")
+        ]
+        for marker in skip_markers:
+            if marker:
+                if marker.name == 'skipif_ocs_version':
+                    skip_condition = marker.args
+                    log.info(skip_condition)
+                    # skip_confition will be a tuple
+                    # and condition will be first element in the tuple
+                    if skipif_ocs_version(skip_condition[0]):
+                        log.info(
+                            f'Test: {item} will be skipped due to {skip_condition}'
+                        )
+                    items.remove(item)
+                    break
+                elif marker.name == 'skipif_lso_deployment':
+                    if skipif_lso_deployment():
```

Comment:
The issue why the travis is failing is that you are trying to call some OC command when pytest is trying to collect the tests. This can break even deployment as well and that's the reason why I've added this --collect-only check in our travis.

The right solution here would be to first check if the cluster is even installed and running. If not you cannot do any call via OC command so you will just continue and skip this check for lso deployment.

If cluster is running you will continue with this check.

Easier solution could be to just wrap this by `try`, `expect` block and if the exception will be raised you will just ignore and skip. But it is more nasty solution. So I prefer more clean one and do as I suggested before.